Sample Applications/CustomComboBox/Styles.xaml

```diff
@@ -145,6 +145,13 @@
 
     <Style x:Key="listBoxStyle" TargetType="ListBox">
         <Setter Property="Foreground" Value="#AEAEAE"/>
+        <Setter Property="ItemContainerStyle">
+            <Setter.Value>
+                <Style>
+                    <Setter Property="AutomationProperties.Name" Value="{Binding Title}"/>
+                </Style>
+            </Setter.Value>
+        </Setter>
```

Comment:
Declaring this here makes it apply to every ListBox in the application.  That works for this sample, but doesn't generalize to apps that have ListBoxes bound to different datasets - each of them needs a different property in the binding path.

Instead, set the ItemContainerStyle at the same place you set ItemsSource - typically directly on each instance of ListBox.  It's OK (even preferred) to set it via StaticResource reference, putting the details of the declaration in a shared location.  In other words, add 
```
     <Style x:Key="MovieItemContainerStyle">
          <Setter Property="AutomationProperties.Name" Value="{Binding Title}"/>
     </Style>
```
to an appropriate ResourceDictionary (this one, in Styles.xaml), and add a reference in every place that sets ItemsSource (such as MainWindow.xaml line 21):
```
     ItemsSource="{Binding Movies}"   ItemContainerStyle="{StaticResource MovieItemContainerStyle}"
```

That works for controls derived from ItemsControl.  But this sample uses the custom ``DropList`` control, which doesn't.  So to get this to work, you have to add an ``ItemContainerStyle`` property to ``DropList`` and propagate it to the underlying ListBox, similar to how ItemsSource declared in DropList.cs and propagated at line 332 of Styles.xaml:
```
          ItemsSource="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=ItemsSource}"
```

A meta-remark about this sample (not about your change):
I don't know what this sample is trying to illustrate.  It's a poor sample of customizing a combo box - the choice of deriving from ContentControl (instead of ItemsControl) means that it doesn't inherit all kinds of useful behavior, including just about everything having to do with automation and accessibility, and thus is responsible for supplying it explicitly.  Which this control fails to do.